### PR TITLE
Feature/shop surfboards

### PR DESCRIPTION
**Pull request by gergana95**
Shop-slider section with surfboards

screen shot 2017-07-14 at 4 58 25 pm

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -340,6 +342,135 @@ q:before, q:after{ content: "";}
...
+    content: "";
+    display: block;
+    position: absolute;
+    width: 250px;
+    height: 320px;
+    background-color: #e8e8e8;
+    z-index: -1;
+    bottom: 0;
+    left: 0;
+    bottom: 0;
```
Comment (superKalo) [inline index.css]: Hm, `bottom: 0` is defined twice?

File: index.html [hunk 2]
```diff
@@ -94,14 +94,71 @@ <h1 class="primary-slider__title font-secondary">Beautiful Escape</h1>
...
+            </div>
+
+            <div class="shop-grid__col-02 left">
+                <img class="catalog__surf-image" src="images/surf-2.png"  alt="slider second board" />
+                <div class="catalog__content">
+                     <span class="content__subtitle font-tertiary">Surfboards</span>
+                     <h3 class="content__title font-secondary">Emery NEM XF</h3>
+
+                     <span class="content__price font-tertiary"">$950</span>
+                     <span class="content__buy font-primary">buy</span>
```
Comment (superKalo) [inline index.html]: Maybe an anchor tag, instead of a span?

File: index.css (hunk 1 above)
Comment (gergana95) [reply index.css]: done!

File: index.html (hunk 2 above)
Comment (gergana95) [reply index.html]: done

File: index.html [hunk 3]
```diff
@@ -94,14 +94,71 @@ <h1 class="primary-slider__title font-secondary">Beautiful Escape</h1>
...
+            </div>
+
+        </div>
+
+            <a class="primary-button font-primary" href="#">show all</a>
+
+            <div class="shop-controls">
+                <svg class="shop-controls__previous-arrow" xmlns="http://www.w3.org/2000/svg" width="11" height="19" viewBox="0 0 11 19">
+                 <path fill="#333" d="M.15 9.64l8.96-8.49c.2-.2.55-.2.76 0 .2.2.2.52 0 .72L1.3 10l8.57 8.13c.2.2.2.52 0 .72-.1.1-.25.15-.38.15a.55.55 0 0 1-.38-.15L.16 10.36a.5.5 0 0 1 0-.72z"/>
+                </svg>
```
Comment (superKalo) [inline index.html]: Probably the parent `.shop-controls` div is unnecessary here. Plus, try to use a <button> tag (with the SVG of the arrows inside).

File: index.html [hunk 4]
```diff
@@ -94,14 +94,71 @@ <h1 class="primary-slider__title font-secondary">Beautiful Escape</h1>
...
+            </div>
+
+            <div class="shop-grid__col-03 left">
+                <img class="catalog__surf-image" src="images/surf-3.png"  alt="slider third board" />
+                <div class="catalog__content">
+                     <span class="content__subtitle font-tertiary">Funboards</span>
+                     <h3 class="content__title font-secondary">Agency GROM</h3>
+
+                     <span class="content__price font-tertiary"">$670</span>
+                     <a class="content__buy font-primary">buy</a>
```
Comment (superKalo) [inline index.html]: Missing `href` attribute on the 3 anchor tags above.

**Comment by superKalo** [comment]
The distance of the left and the right arrow from the page wrapper is not even. See:

screenshot at jul 14 18-28-50

File: index.css [hunk 5]
```diff
@@ -340,6 +339,146 @@ q:before, q:after{ content: "";}
...
+.shop-container button {
+    position: relative;
+    border-color: transparent;
+    background-color: transparent;
+}
+
+.shop-controls__previous-arrow {
+    position: absolute;
+    bottom: 250px;
+    left: -599px;
```
Comment (superKalo) [inline index.css]: The `left` looks like a magic (positioning) number to me :-)

Move the `.shop-controls__previous-arrow` properties (absolute, left, bottom) to the `<button>` element. Then - your issue will be resolved and you can position it easily.

Same applies to the `.shop-controls__next-arrow`'s `right` magic number.

**Comment by superKalo** [comment]
Okay, now it looks awesome! 😎 🚀 🌻 

You can merge it 👍

**Pull request merged**